server/game/cards/attachments/04/beggarking.js

```diff
@@ -9,10 +9,10 @@ class BeggarKing extends DrawCard {
         });
         this.reaction({
             when: {
-                onPlotRevealCompleted: () => (
-                    _.any(this.game.getPlayers(), player => (
-                        player !== this.controller &&
-                        this.controller.activePlot.getIncome(true) < player.activePlot.getIncome(true)
+                onPlotsRevealed: event => (
```

Comment:
This logic is a bit clumsy now, although it will be nice for multi-player support if we ever do it, it's now checking all plots and ignoring one of them, rather than just getting the other player's plot and comparing that directly, if that makes sense?